examples/pipeline_wavernn/processing.py

```diff
@@ -31,15 +31,19 @@ class NormalizeDB(nn.Module):
     r"""Normalize the spectrogram with a minimum db value
     """
 
-    def __init__(self, min_level_db):
+    def __init__(self, min_level_db, normalization):
         super().__init__()
         self.min_level_db = min_level_db
+        self.normalization = normalization
 
     def forward(self, specgram):
-        specgram = 20 * torch.log10(torch.clamp(specgram, min=1e-5))
-        return torch.clamp(
-            (self.min_level_db - specgram) / self.min_level_db, min=0, max=1
-        )
+        if self.normalization:
+            specgram = 20 * torch.log10(torch.clamp(specgram, min=1e-5))
+            return torch.clamp(
+                (self.min_level_db - specgram) / self.min_level_db, min=0, max=1
+            )
+        else:
+            return torch.log10(torch.clamp(specgram, min=1e-5))
```

Comment:
Is the factor of 20 missing intentional?

If so, let's refactor this as:
```python
        specgram = torch.log10(torch.clamp(specgram, min=1e-5))
        if self.normalization:
            specgram = torch.clamp(
                (self.min_level_db - 20 * specgram) / self.min_level_db, min=0, max=1
            )
        return specgram
```

Reply:
yes, we don't need the factor of 20 when `normalization` is false.
I agree, will update it.